inlong-manager/manager-plugins/base/src/main/java/org/apache/inlong/manager/plugin/listener/StartupSortListener.java

```diff
@@ -98,6 +99,14 @@ public ListenerResult listen(WorkflowContext context) throws Exception {
         }
 
         for (InlongStreamInfo streamInfo : streamInfos) {
+            boolean isRealTimeSync = InlongConstants.DATASYNC_REALTIME_MODE
+                    .equals(groupResourceForm.getGroupInfo().getInlongGroupMode());
+            // do not submit flink job if the group mode is offline and the stream is not config successfully
+            if (isRealTimeSync && !StreamStatus.CONFIG_SUCCESSFUL.getCode().equals(streamInfo.getStatus())) {
```

Comment:
`streamInfo.getStatus()` is always `StreamStatus.CONFIG_ING`.

Reply:
fixed, PTAL